SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -20,7 +20,20 @@
 DEFINE_LOG_CATEGORY(LogSpatialDeploymentManager);
 
 static const FString SpatialExe(TEXT("spatial.exe"));
-static const FString SpatialServiceVersion(TEXT("20190716.094149.1b6d448edd"));
+static const FString SpatialServiceVersion(TEXT("20191128.003423.475a3c1edb"));
+
+namespace
+{
+	FString GetDomainEnvironmentStr(bool bIsInChina)
```

Comment:
Wonder if an Enum would be better suited for this. User's could set an override using the enum like a dropdown from the settings menu. If it's unchanged it would be default. If the domain override was "CN" it would use the correct domain and env for China. It would also make it nicer to not pass around a bool "bIsInChina".

Reply:
Yeah, this is the Editor<->Services problem - we can't directly read from the settings whether we are in a China environment :(

Reply:
So it turns out that isn't a thing, another option is to query the setting on the entry points to SpatialGDKEditorCloudLauncher? We call this with an async callback though which can't have arguments so we might end up setting it before that call anyway - thoughts? I prefer the restart because it ensures any new code won't need to remember to set it. 

This may get fixed once we add the services settings so perhaps we could do this in the dev branch?

Reply:
"Wonder if an Enum would be better suited for this. User's could set an override using the enum like a dropdown from the settings menu"

What could these enums be? For most things in the west the outcome is the same - what about West and China?

Reply:
Regarding the services setting yeah it seems like we could fix it there.
Regarding the enum, could just be 'default' and 'CN' :P

Reply:
Cool, resolved this part in the latest commit!